doc/source/whatsnew/v1.3.0.rst

```diff
@@ -247,7 +247,7 @@ These are bug fixes that might have notable behavior changes.
 ``Categorical.unique`` now always maintains same dtype as original
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Previously, when calling :meth:`~Categorical.unique` with categorical data, unused categories in the new array
+Previously, when calling :meth:`Categorical.unique` with categorical data, unused categories in the new array
```

Comment:
this link broken anyway

Reply:
Right, but the difference is that ``:meth:`~Categorical.unique`` just shows "unique" whereas the change has it show up as "Categorical.unique". I think the latter is preferred.

Reply:
didn't mean for this to be changed. feel free to change this again. but was thinking that could maybe reword instead.

Reply:
Doh! I changed it just to make sure my understanding of the behavior was correct, but forgot to change it back. The fix is now in #41822